src/web/core/src/main/java/org/geoserver/web/data/layergroup/RootLayerEntryPanel.java

```diff
@@ -39,7 +39,11 @@ public RootLayerEntryPanel(String id, final Form form, WorkspaceInfo workspace)
         final TextField<LayerInfo> rootLayerField = new TextField<LayerInfo>("rootLayer") {
             @Override
             public IConverter getConverter(Class<?> type) { 
-                return form.getConverter(type);
+                if (LayerInfo.class.isAssignableFrom(type)) {
```

Comment:
The form in question doesn't exist any longer (because the LayerGroupEditPage was refactored), and the converter creation code was moved to a more appropriate place. However, I guess the "if" construction has become obsolete